tests/test_timeline.py

```diff
@@ -47,6 +51,12 @@ def test_metadata(self):
         self.assertIsOTIOEquivalentTo(tl, decoded)
         self.assertEqual(tl.metadata, decoded.metadata)
 
+    def test_big_integers(self):
+        result = otio.adapters.read_from_file(BIG_INT_TEST)
+
+        self.assertTrue(result.tracks[0][0].metadata["foobar"]["toobig"] > 0)
+        self.assertTrue(result.tracks[0][0].metadata["foobar"]["verybig"] > 0)
+
```

Comment:
here, we could add a test for setting an uint64_t to (0x7FFFFFFFFFFFFFFF + N) and verifying that the value is (N - 1)

Reply:
It seems important to check the exact value that we get, not just `> 0`.
Also, can we check that large negative numbers come through as expected?